src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigatorContext.java

```diff
@@ -69,6 +76,21 @@ public String getTeamSlug() {
         return teamSlug;
     }
 
+    /**
+     * Sets the topics which the repositories must have.
+     */
+    void setTopics(ArrayList<String> topics) {
+        this.topics = topics;
+    }
+
+    /**
+     * Gets the topics which the repositories must have.
+     * @return topics
+     */
+    public ArrayList<String> getTopics() {
```

Comment:
should the interface not the implementation
```suggestion
    public List<String> getTopics() {
```